advanced_start/more.rst

```diff
@@ -0,0 +1,159 @@
+
+.. code :: python
+
+    def __init__(self):
+        train = pd.read_csv("train.csv")
+        
+        self.cols = ['petal_length', 'petal_width', 'sepal_length', 'sepal_width'] 
+        colsRes = ['class']
+        trainArr = train.as_matrix(self.cols) 
+        trainRes = train.as_matrix(colsRes) 
+        
+        rf = RandomForestClassifier(n_estimators=100)
+        rf.fit(trainArr, trainRes)
+        self.rf = rf
+
+Now we can write our ``predict`` function. The second argument, ``inputArr``, is the input -- in this case, our four ``Double``\s. To return text, we simply return from the function in Python.
+
+.. code :: python
+
+    def predict(self, inputArr):
+        points = [inputArr]
+        df = pd.DataFrame(points, columns=self.cols)
+
+        results = self.rf.predict(df)
+        return results.item()
+
+Configuring your module
+***********************
+
+When your module is started, it is run in a Linux container on the NStack server. Because our module uses libraries like ``pandas`` and ``sklearn``, we have to tell NStack to install some extra operating system libraries inside your module's container. NStack lets us specify these in our ``nstack.yaml``` configuration file in the ``packages`` section. Let's add the following packages:
+
+.. code :: yaml
+
+    packages: ['numpy', 'python3-scikit-learn', 'scipy', 'python3-scikit-image', 'python3-pandas']
+
+Additionally, we want to tell NStack to copy our ``train.csv`` file into our module, so we can use it to train our data. ``nstack.yaml`` also has a section for specifying files you'd like to include:
+
+.. code :: yaml
+
+    files: ['train.csv']
+
+Publishing and starting
+***********************
+
+Now we're ready to build and publish our classifier. Remember, even though we run this command locally, our module gets built and published to your NStack Server.
+
+.. code :: bash
+
+    ~/Irisclassify/ $ nstack build
+    Building NStack Container module Irisclassify. Please wait. This may take some time.
+    Module Irisclassify built successfully. Use `nstack list functions` to see all available functions.
+
+We can now see ``irisclassify.predict`` in the list of existing functions (along with previously built functions like `demo.numChars`) by running the suggested command
+
+.. code :: bash
+ 
+   ~/irisclassify/ $ nstack list functions
+    Irisclassify.predict : (Double, Double, Double, Double) -> Text
+    Demo.numChars : Text -> Integer
+
+Our classifier is now published, but to use it we need to connect it to an event source and sink. In the previous tutorial, we used HTTP as a source, and the NStack log as a sink. We can do the same here by starting the following workflow.
+
+.. code :: bash
+   
+    ~/Irisclassify/ $ nstack start "sources.http<(Double, Double, Double, Double)> { http_path : "/irisendpoint" } | irisclassify.predict | sinks.log<Text>"
```

Comment:
This is sadly still not possible. You need a complete workflow for `nstack start` which could be referenced unqualified. The current example has a workflow and a project yaml, which could be recreated?